src/Sulu/Bundle/AudienceTargetingBundle/Entity/TargetGroupRepository.php

```diff
@@ -73,6 +73,18 @@ public function save(TargetGroupInterface $targetGroup)
     /**
      * {@inheritdoc}
      */
+    public function findByIds($ids)
+    {
+        $query = $this->createQueryBuilder('targetGroup')
```

Comment:
I am not sure about this query... It is only needed to return something in the `getContentData` method of the content type. I didn't do the joins here, because in most of the cases we probably don't even need the target group in the template itself anyway.

Question is if this query should be executed at all, or we only return an empty array in the content type.